### PR TITLE
Edit README.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
skip ci  please
Why don't you `code fence` the `skip ci` instructions, it would be much easier to explain/update.
- Please add description for changes you are making.
Edit of read me

@tdystra please review

### CODE REVIEW

**Comment by nguerrera** [comment]
These are good changes but now out-of-date with 2.0 being RTM as well. @johnbeisner can you help get the conflicts resolved and the text updated to reflect that 2.0 is RTM.

**Comment by johnbeisner** [comment]
@dotnet-bot test Debian8.2 x64 Debug please

File: README.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
+This repo contains the source code for cross-platform [.NET Core](http://github.com/dotnet/core) command line toolchain. It contains the implementation of each command, the native packages for various supported platforms and the documentation.
 
 Looking for V1 of the .NET Core tooling?
 ----------------------------------------
 
-If you are looking for the v1.0.1 release of the .NET Core tools (CLI, MSBuild and the new csproj), head over to https://dot.net/core and download!
+If you are looking for the v1.0.1 release of the .NET Core tools (CLI, MSBuild and the new csproj), see https://dot.net/core.
 
-> **Note:** the release/2.0.0 branch of the CLI repo is based on the upcoming v2 of .NET Core and is considered pre-release. For production-level usage, use the
+> **Note:** the release/2.0.0 branch of the CLI repo is based on the upcoming v2 of .NET Core and is considered pre-release. For production-level usage, use
```
Comment (nguerrera) [inline README.md]: This needs to be changed to indicate that the release/2.0.0 branch is based on the upcoming 2.0.x servicing release. 2.0 is RTM and can be obtained from https://dot.net/core.

Comment (nguerrera) [reply README.md]: @johnbeisner Please make this change as a follow-up and make sure the text in master is correct about master being v2.1.

**Pull request merged**